Fast and Accurate GPU Quantization for Transformers.md

```diff
@@ -93,11 +92,7 @@ To perform calibration, we used TensorRT’s [PyTorch Quantization Toolkit](http
 
 A final distinction to be made is how quantization parameters are shared between elements of our parameters and activations. Throughout this blog, we'll use the following diagram to illustrate a matmul:
```

Comment:
We don't really use it throughout the blog? Do we wanna re-word?

Reply:
Oh I think I get what is meant now, we use W, X, Y later on. We might want to remove this comment anyway if we are planning to move this section after Specifics of INT8 GEMMs (where the equations are defined).

Reply:
Can we change line 99 from `It’s also feasible to share quantization parameters` to `It’s also common to share quantization parameters`

Reply:
Have gone with both of these